_health-care/apply.md

```diff
@@ -1,61 +1,74 @@
 ---
 title: Apply for Health Care
-template: 2-detailed-landing
 ---
 
+<div class="main" role="main" markdown="0">
+
+<div class="va-action-bar--header">
+  <div class="row">
+    <div class="small-12 columns">
+      <a class="usa-button-primary va-button-primary" href="#">Apply for VA Health Care</a>
+    </div>
+  </div>
+</div>
+
 <div class="section one" markdown="0">
 
+
+
 <div class="primary" markdown="0">
 <div class="row" markdown="0">
-<div class="small-12 columns" markdown="1">
+<div class="small-12 columns usa-content" markdown="1">
+<div markdown="1">
+The VA operates America’s largest integrated health care system, with over 700 sites of care. It is consistently ranked among the nation’s top health care providers.
 
-### Who should read this
-Veterans seeking health care. 
+Health care at the VA includes preventive, primary, and specialty care; prescriptions; mental health care; home health care; geriatrics and extended care; medical equipment and prosthetics; and more. Most Veterans qualify for cost-free health care services, although some Veterans must pay modest copays for health care or prescriptions.
```

Comment:
I know this was from the original content, but there shouldn't be semicolons separating the list items, just commas.